parsers/UY.py

```diff
@@ -149,11 +153,10 @@ def fetch_consumption(
 ) -> list:
     """collects consumption data from ADME and format all data points for target_datetime"""
     if target_datetime is None:
-        target_datetime = arrow.utcnow().replace(tzinfo=UY_TZ).datetime
+        target_datetime = datetime.now().replace(tzinfo=UY_TZ)
```

Comment:
Tried to get rid of arrow, but could not replace all of it, as I don't know what's the format of the string returned by the source where `arrow.get` is used

Reply:
I'll be taking a look at our usage of arrow and removing it the next days so hopefully I can remove the last bits then.

Reply:
Nice! But FYI here we won't be able to get rid of it until the source comes back online